samples/test-runner/build.gradle.kts

```diff
@@ -58,6 +58,15 @@ instrumentation {
             cpuCoresLimit = "1.3",
             memoryLimit = "4Gi"
         )
```

Comment:
Didn't rebuild older images.
I've checked locally that changes in Docker didn't affect them: qemu version is the same, works locally.